alphadia/workflow/optimization.py

```diff
@@ -165,6 +157,17 @@ def step(
                 verbosity="progress",
             )
 
+    def skip(self):
+        """Increments the internal counter for the number of consecutive skips and checks if the optimization should be stopped."""
+        self.num_consecutive_skips += 1
+        if self._unnecessary_to_continue:
```

Comment:
`_unnecessary_to_continue` is a bit sloppy, mb find a better name for this concept?

Reply:
Yes, I agree. I tried for a while and couldn't come up with anything, but I will try again.

Maybe ``_batch_substantially_bigger``?

Reply:
mb also add some of the PR description here in order to fully get this

Reply:
Or ``_large_drop_in_precursor_proportion``?